test/eth.accounts.signTransaction.js

```diff
@@ -24,6 +24,63 @@ var commonBerlin = {
     hardfork: 'berlin'
 };
 
+var commonLondon = {
```

Comment:
I'd recommend using `let` instead of `var` to declare variables. `let` is a more modern syntax than `var` and it aims to resolve some weird hoisting behaviors that `var` has that could lead to bugs.

Reply:
Hey @kchia, thank you for the input! Unfortunately most of Web3.js was written a while back, before modern JavaScript techniques were widely available/supported. We're currently working on a `4.x` version this uses TypeScript and modern JS features all the way up to ES2020. So for `1.x` and `3.x` the concern is to limit the amount of changes made to reduce introducing any new bugs